addons/l10n_in_ewaybill_stock/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name": """E-waybill Stock""",
+    "version": "1.0",
+    "category": "Accounting/Localizations/EDI",
+    "depends": [
+        "stock", "l10n_in_edi_ewaybill",
```

Comment:
@hamo-odoo We should depend on l10n_in_stock here.  That way we can define a method in l10n_in_stock to get the price/tax information from stock.move.   That way we can inherit that method in the existing l10n_in_purchase_stock and l10n_in_sale_stock and we do not need the l10n_in_ewaybill_stock_purchase/sale modules.  (we just use the already existing ones)